src/main/resources/assets/react/documentTypes/AddOrEditLocalFieldModal.jsx

```diff
@@ -193,9 +193,7 @@ export const AddOrEditLocalFieldModal = ({
 							// labelPosition= {'right'}
 							toggle
 							onChange={()=>{
-								/* Currently ignored */
-								//setActive(!active)
-								console.log('Create active state');
+								setActive(!active);
```

Comment:
I guess this should work, in that you negate the previous state. I usually get the new value from the onChange event.